src/store/tournaments/store.ts

```diff
@@ -0,0 +1,46 @@
+import { ITournament, TournamentsState } from "./types";
+import { Map } from "../admin/maps/types";
+import TournamentsService from "@/services/TournamentsService";
+import MapsService from "@/services/MapsService";
+import { defineStore } from "pinia";
+
+export const useTournamentsStore = defineStore("tournaments", {
+  state: (): TournamentsState => ({
+    // loaded: false,
```

Comment:
It is intended because it not part of the type definition and will throw an error:
https://github.com/w3champions/website/blob/7d752990949f38879ce1d75ddf83b37ace9a6707/src/store/tournaments/types.ts#L35-L39

So the `fixme` notice is missing 😉

Reply:
Oh okay. Maybe we can just delete that property then. But we can do that in another pr. merging!